easybuild/easyblocks/i/ifort.py

```diff
@@ -76,3 +76,14 @@ def sanity_check_step(self):
         custom_commands = ["which ifort"]
 
         IntelBase.sanity_check_step(self, custom_paths=custom_paths, custom_commands=custom_commands)
+
+    def make_module_req_guess(self):
+        """
+        Additional paths to consider for prepend-paths statements in module file
+        """
+        guesses = super(EB_icc, self).make_module_req_guess()
+        if LooseVersion(self.version) >= LooseVersion('2016'):
+            # This enables the creation of fortran 2008 bindings in MPI
+            guesses['CPATH'].extend(['include'])
```

Comment:
@damianam: Would it make sense to just add `include` in the initial `CPATH` list in `EB_icc.make_module_req_guess` instead?

`<prefix>/include` will only be added to `$CPATH` if it exists, so it's OK if it's also considered for `icc`

Also, style nitpicking: to add a single element, you should use `append(x)` rather than `extend([x])` ;-)

Reply:
No, in `$EBROOTICC/include` there are some other headers, like `complex.h`, that apparently cause problems. Besides that, there is something fishy, the resulting module now is missing some paths in other variables. I'll try to take a look at it today.

Reply:
OK, thanks for clarifying, makes sense to only do this for `ifort` then.

I see no reason why other paths would suddenly go missing though...